lib/praxis/action_definition.rb

```diff
@@ -87,7 +110,14 @@ def params(type=Attributor::Struct, **opts, &block)
         update_attribute(@params, opts, block)
       else
         @params = create_attribute(type, **opts, &block)
+        if (api_info = ApiDefinition.instance.info(resource_definition.version))
+          if api_info.base_params
+            @params.attributes.merge!(api_info.base_params.attributes)
```

Comment:
wouldn't this override the wrong way?
I think that if you have a `foobar` in the base_params ... and the action has a `foobar` too... the one in the action should take precedence...no?